README.md

```diff
@@ -7,37 +7,33 @@
 
 An Agent acts on behalf of one or more clients. To do so they require a valid relationship to be in place. 
 As Agents move to MDTP, our intention where possible is that their Agent/Client relationships that exist in HoDs are honoured. 
+
 The Mapping journey has arisen due to the need to honour SA relationships that are stored within CESA. 
```

Comment:
Minor point: This line, and the paragraph below, are specific to just SA relationships, but it's starting to look a little dated now that we capture all known old-world agent enrolment identifiers.